examples/rgbpp/README.md

```diff
@@ -25,6 +25,44 @@ But when you're searching for this transaction in [Bitcoin Core](https://bitcoin
 018025fb6989eed484774170eefa2bef1074b0c24537f992a64dbc138277bc4a
 ```
 
+## Examples with Queue service(Recommended)
+
+### Mint XUDT
+As a simple example, Omiga protocol is reused for a quick demonstration of XUDT asset insurance.
+Developers have the option to utilize an existing XUDT (User-Defined Token) on CKB.
+
+```shell
+npx ts-node examples/rgbpp/queue/1-mint-xudt.ts 
+```
+
+### Jump XUDT from CKB to BTC
+
+```shell
+npx ts-node examples/rgbpp/queue/2-ckb-jump-btc.ts 
+```
+
+### Transfer RGB++ asset on BTC
+
+```shell
+npx ts-node examples/rgbpp/queue/3-btc-transfer.ts 
+```
+
+### Jump RGB++ asset from BTC to CKB
+
+```shell
+npx ts-node examples/rgbpp/queue/4-btc-jump-ckb.ts 
+```
+
+### Unlock BTC time cells on CKB
+
+**Warning: Wait at least 6 BTC confirmation blocks to unlock the BTC time cells after 4-btc-jump-ckb.ts**
+
+```shell
+npx ts-node examples/rgbpp/queue/5-spend-btc-time-cell.ts 
+```
+
```

Comment:
I think the step of unlocking btc-time-lock cell is automatically handled by the RGB++ Quere service.

Reply:
YES. It's an error description and I will remove it